lib/winston-loggly.js

```diff
@@ -58,7 +58,7 @@ var Loggly = exports.Loggly = function (options) {
     bufferOptions: options.bufferOptions || {size: 500, retriesInMilliSeconds: 30 * 1000}
   });
 
-  this.timestamp = options.timestamp || false;
```

Comment:
Thanks @mchaudhary for the feedback.

I have updated the condition to set the timestamp by default and also to set it to false if user specify it explicitly.

**I've covered below conditions,:**

| User's Input explicitly | Timestamp field in the log event |
| --- | --- |
| true | Yes |
| false | No |
| Not set | Yes |
| anything except `false` boolean | Yes |